lib/net.js

```diff
@@ -1163,11 +1163,11 @@ var createServerHandle = exports._createServerHandle =
     debug('bind to ' + (address || 'anycast'));
     if (!address) {
       // Try binding to ipv6 first
-      err = handle.bind6('::', port);
+      err = handle.bind6('::1', port);
       if (err) {
         handle.close();
         // Fallback to ipv4
-        return createServerHandle('0.0.0.0', port);
+        return createServerHandle('localhost', port);
```

Comment:
I find 'localhost' to be more expressive to users but technically 127.0.0.1 is slightly more efficient.